src/default-settings.js

```diff
@@ -8,6 +8,7 @@ const settings = {
     endy_col: 'AENDY',
     sev_col: 'AESEV',
     rel_col: 'AEREL',
+    ser_col: 'AESER',
```

Comment:
Is `AESER` required now? If so, does this break backwards compatibility with v1.2.0?